tests/perf/perf_tests.cc

```diff
@@ -376,7 +390,13 @@ void run_all(const std::vector<std::string>& tests, const config& conf)
         return tests.empty() || it != tests.end();
     };
 
+    size_t max_name_column_length = 0;
+    for (auto&& test : all_tests() | boost::adaptors::filtered(can_run)) {
+        max_name_column_length = std::max(max_name_column_length, test->name().size());
+    }
```

Comment:
how about using `std::ranges::max_element()` ? more à la mode this way. as we have the luxury of using C++20 now..

Reply:
I was using the non-ranges version at first but it's not actually much nicer or shorter as it returns an iterator so you still kind of have to do a check afterwards whether the iterator is valid (i.e.: empty range).